src/antsibull_docs/write_docs.py

```diff
@@ -630,6 +667,8 @@ async def write_plugin_lists(collection_name: str,
 
     await write_file(index_file, index_contents)
 
+    flog.debug('Leave')
+
```

Comment:
Did you mean to leave this in here?

Reply:
Yes, since the function now logs that it was entered, it should also log when it is left.